homeassistant/components/homekit/type_media_players.py

```diff
@@ -394,12 +389,12 @@ def set_remote_key(self, value):
                 service = SERVICE_MEDIA_PLAY_PAUSE
             params = {ATTR_ENTITY_ID: self.entity_id}
             self.call_service(DOMAIN, service, params)
-        else:
-            # Other keys can be handled by listening to the event bus
-            self.hass.bus.fire(
-                EVENT_HOMEKIT_TV_REMOTE_KEY_PRESSED,
-                {ATTR_KEY_NAME: key_name, ATTR_ENTITY_ID: self.entity_id},
-            )
+
+        # All keys can be also handled by listening to the event bus
```

Comment:
Fire for all events, including play/pause, in case the underlying media player doesn't implement it, but a user would like to wire it up some other way

Reply:
What use case are you envisioning for firing the key press event for play/pause if the device supports it?

Should we keep the `if features & (SUPPORT_PLAY | SUPPORT_PAUSE):` check and set `self._supports_play_pause` and change the if to `if key_name == KEY_PLAY_PAUSE and self._supports_play_pause`?

Reply:
Good point! Will update thi